### PR TITLE
Update stream-processing.md

### PR DESCRIPTION
**Pull request by lionelpenuchot**
the following paragraph need to be updated according to new ASA capabilities https://learn.microsoft.com/en-us/azure/architecture/data-guide/technology-choices/stream-processing#integration-capabilities

### CODE REVIEW

**Comment by Court72** [comment]
@martinekuan 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by martinekuan** [comment]
#sign-off

File: docs/data-guide/technology-choices/stream-processing.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Choose a stream processing technology
 description: Compare options for real-time message stream processing in Azure, with key selection criteria and a capability matrix.
 author: martinekuan
-ms.author: architectures
+ms.author: lionelp
```
Comment (Jak-MS) [inline docs/data-guide/technology-choices/stream-processing.md:5]:
```suggestion
ms.author: architectures
```

**Comment by Jak-MS** [comment]
@lionelpenuchot - we don't allow metadata changes in the public repo (so I changed it back to the original). That said, you can make that change by submitting a PR to the private repo if @martinekuan is in agreement with that change. thanks.

ADLS should be spelled out. (I've merged it as is, but we recommend you spell it out.)

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**